packages/web/src/initialize.ts

```diff
@@ -180,6 +193,9 @@ const initSingleTenancy = async () => {
                     name: SINGLE_TENANT_ORG_NAME,
                     domain: SINGLE_TENANT_ORG_DOMAIN,
                     inviteLinkId: crypto.randomUUID(),
+                    memberApprovalRequired: env.FORCE_MEMBER_APPROVAL_REQUIRED === 'true' ? true : 
+                                          env.FORCE_MEMBER_APPROVAL_REQUIRED === 'false' ? false : 
+                                          true, // default to true if FORCE_MEMBER_APPROVAL_REQUIRED is not set
```

Comment:
```suggestion
    ...(env.FORCE_MEMBER_APPROVAL_REQUIRED === 'true' && {
      memberApprovalRequired: true,
    })
```

default values for data in the db are set by the schema so its best to not have them set in the code. We only want to set this if the env var is true anyways